server/playground.py

```diff
@@ -699,6 +700,73 @@ async def hangle_bigquery_post_request():
     }
 
 
+def _mysql_connection() -> pymysql.Connection:
+    return pymysql.connect(
+        host=os.environ["MYSQL_HOST"],
+        user=os.environ["MYSQL_USER"],
+        password=os.environ["MYSQL_PASSWORD"],
+        database=os.environ["MYSQL_DATABASE"],
+    )
+
+
+### MySQL back-end routes
+@app.get("/mysql")
+async def handle_mysql_get_request():
+    with _mysql_connection().cursor() as cursor:
+        cursor.execute(
+            "SELECT t.table_name FROM information_schema.tables t "
+            f"WHERE t.table_type = 'BASE TABLE' AND t.table_schema = '{os.environ['MYSQL_DATABASE']}'"
+        )
+        tables = [table[0] for table in cursor.fetchall()]
+
+    return jsonify(tables)
+
+
+def _mysql_table_info() -> dict[str, list[str]]:
+    return {
+        row["table_name"]: json.loads(row["column_names"])
+        for _, row in pd.read_sql(
+            """SELECT t.table_name "table_name", JSON_ARRAYAGG(c.column_name) "column_names" """
+            "FROM information_schema.tables t "
+            "INNER JOIN information_schema.columns c ON t.table_name = c.table_name "
+            "WHERE t.table_type = 'BASE TABLE' AND t.table_schema = 'playground_db' "
+            "GROUP BY t.table_name "
+            "ORDER BY t.table_name;",
+            _mysql_connection(),
```

Comment:
```suggestion
        for _, row in pd.read_sql(
            """SELECT t.table_name table_name, JSON_ARRAYAGG(c.column_name) column_names
            FROM information_schema.tables t 
            INNER JOIN information_schema.columns c ON t.table_name = c.table_name 
            WHERE t.table_type = 'BASE TABLE' AND t.table_schema = 'playground_db' 
            GROUP BY t.table_name 
            ORDER BY t.table_name;""",
            _mysql_connection(),
```
much clear ? WDYT ?

Reply:
Don't like it, looks bad when printed. And I hate left-indented multiline strings :smile: